financial-connections-core/build.gradle

```diff
@@ -0,0 +1,52 @@
+apply from: configs.androidLibrary
+
+apply plugin: 'checkstyle'
+apply plugin: 'org.jetbrains.kotlin.plugin.parcelize'
+apply plugin: 'kotlinx-serialization'
+apply plugin: 'com.google.devtools.ksp'
+
+android {
+    defaultConfig {
+        testApplicationId "com.stripe.android.financialconnections.test"
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    }
+    testOptions {
+        unitTests {
+            includeAndroidResources = true
+        }
+    }
+}
+
+dependencies {
+    api project(":stripe-core")
+    api project(":payments-model")
```

Comment:
This is just needed to expose [Token](https://github.com/stripe/stripe-android/blob/01ce6e7d7230c315b29a7accc12476dd495fc610/financial-connections-core/src/main/java/com/stripe/android/financialconnections/model/FinancialConnectionsSession.kt#L78). 

Although it's a light dependency, if we expose this publicly in the future we might want to